docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,267 @@
+
+```json {class="line-numbers linkable-line-numbers"}
+  {
+    "name": "my_color_detector",
+    "type": "vision",
+    "model": "color_detector",
+    "attributes": {
+      "segment_size_px": 100,
+      "detect_color": "#43a1d0",
+      "hue_tolerance_pct": 0.06
+    }
+  },
+  ... // Other services
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Click **Save Config** and head back to the **Builder** mode.
+
+### Data management service to collect images
+
+To enable data capture on your robot, add and configure the [data management service](/services/data/) to capture and store data on your robot's computer.
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. At the bottom-left of the page, click **Create service**.
+1. Choose `Data Management` as the type and name your instance of the data manager `dm`.
+   This service syncs data from your robot to the Viam app in the cloud.
+1. Select **Create**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data every 0.1 minutes in the <file>~/.viam/capture</file> directory.
+
+   You can leave the default settings as they are.
+   Click **Save Config** at the bottom of the window.
+
+   ![An instance of the data management service named "dm". The cloud sync and capturing options are toggled on and the directory is empty. The interval is set to 0.1](/tutorials/pet-photographer/data-management-services.png)
+
+   For more detailed information see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+   {{% /tab %}}
+   {{% tab name="JSON Template" %}}
+   Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "dm",
+      "type": "data_manager",
+      "namespace": "rdk",
+      "attributes": {
+        "sync_interval_mins": 0.1,
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": []
+      }
+    },
+  ... // Vision and other services
+```
+
+Click **Save Config** and head back to the **Builder** mode.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Configure the color filter camera
+
+Before you can interact with the vision and data management services, you must configure your camera to filter color and store photos to Viam's cloud.
+
+### Configure the module's executable
+
+This tutorial makes use of [Viam's color filter example](https://github.com/viam-labs/modular-filter-examples/tree/main/colorfilter). However, you can modify the filter's source code or write your own filter for different components using [this guide](/extend/modular-resources/create/).
+
+Navigate to the `modular-filter-examples/colorfilter/module` directory on your robot's terminal and run the following command:
+
+```{class="command-line" data-prompt="$"}
+go build
```

Comment:
in the set up step you instructed people to install go on your local development computer - should it be installed on the robot's computer?